connector-common/pom.xml

```diff
@@ -84,12 +84,6 @@
             <artifactId>camel-quarkus-seda</artifactId>
         </dependency>
 
-        <!-- Quarkus -->
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-micrometer-registry-prometheus</artifactId>
-        </dependency>
```

Comment:
Will the connectors still expose metrics that can be collected by Prometheus after this is removed?